tsconfig.json

```diff
@@ -9,7 +9,6 @@
     "sourceMap": true
   },
   "include": [
-    "examples/typescript",
```

Comment:
what's the benefit of moving examples out of the default project and creating separated ones?

Reply:
`Post` and `User` are `typeof extends Bone` in basic example, but they are `typeof extends SequelizeBone` in sequelize example, it will throw error  due to `declare module 'egg'`